es2020.md

```diff
@@ -0,0 +1,48 @@
+This document specifies the extensions to the core ESTree AST types to support the ES2020 grammar.
+
+# Literal
+
+```js
+extend interface Literal <: Expression {
+    type: "Literal";
+    value: string | boolean | null | number | RegExp | bigint;
+}
+```
+
+- `value` property can be a `BitInt` value to represent `BitInt` literals
+  such as `100n`.
+
+## BigIntLiteral
+
+```js
+interface BigIntLiteral <: Literal {
+  bigint: string;
+}
+```
+
+- `bigint` property is the string representation of the `BitInt` value.
+  It includes the suffix `n`.
```

Comment:
Why?

Reply:
No reason :D

Just I thought the spec should clear which the value of `bigint` property is.
I don't mind that it includes or doesn't include `n`.

Reply:
I think it's great that the spec is explicit about it. I would prefer to omit the `n`. You just need to be able to reconstruct the value from this field, and there's no point in adding an `n` to the end. Besides, BigInt's `toString` also omits the `n`.

Reply:
Sounds reasonable. I will change this sentence.